06-01 17:50:20 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Tabelle "T_DEVICE2" nicht gefunden
Table "T_DEVICE2" not found; SQL statement:
SELECT * FROM ITP.T_DEVICE2 [42102-170]
07-26 20:25:12 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Syntax Fehler in SQL Befehl "DESC[*] ITP.T_DEVICE "; erwartet "DELETE, DROP, DECLARE, DEALLOCATE, {"
Syntax error in SQL statement "DESC[*] ITP.T_DEVICE "; expected "DELETE, DROP, DECLARE, DEALLOCATE, {"; SQL statement:
desc ITP.T_DEVICE [42001-170]
07-26 20:30:52 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Syntax Fehler in SQL Befehl "DROP T_DEVICE"; erwartet "TABLE, INDEX, USER, SEQUENCE, CONSTANT, TRIGGER, VIEW, ROLE, ALIAS, SCHEMA, ALL, DOMAIN, TYPE, DATATYPE, AGGREGATE"
Syntax error in SQL statement "DROP T_DEVICE"; expected "TABLE, INDEX, USER, SEQUENCE, CONSTANT, TRIGGER, VIEW, ROLE, ALIAS, SCHEMA, ALL, DOMAIN, TYPE, DATATYPE, AGGREGATE"; SQL statement:
DROP T_DEVICE [42001-170]
07-26 20:30:58 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Syntax Fehler in SQL Befehl "DROP ITP[*].T_DEVICE"; erwartet "TABLE, INDEX, USER, SEQUENCE, CONSTANT, TRIGGER, VIEW, ROLE, ALIAS, SCHEMA, ALL, DOMAIN, TYPE, DATATYPE, AGGREGATE"
Syntax error in SQL statement "DROP ITP[*].T_DEVICE"; expected "TABLE, INDEX, USER, SEQUENCE, CONSTANT, TRIGGER, VIEW, ROLE, ALIAS, SCHEMA, ALL, DOMAIN, TYPE, DATATYPE, AGGREGATE"; SQL statement:
DROP ITP.T_DEVICE [42001-170]
07-26 20:38:35 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Tabelle "T_DEVICE" nicht gefunden
Table "T_DEVICE" not found; SQL statement:
DROP TABLE ITP.T_DEVICE [42102-170]
07-26 20:39:26 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Syntax Fehler in SQL Befehl "CREATE TABLE ITP.T_DEVICE
( 
	PK_DEVICEID INT PRIMARY KEY NOT NULL, 
	IDENTIFIER INT NOT NULL, 
	ACTIVITY DATE NOT NULL, 
	LATITUDE DATE NOT NULL, 
	LONGITUDE DATE NOT NULL 
)  

ALTER[*] TABLE ITP.T_DEVICE ADD CONSTRAINT IDENTIFIER_UNIQUE UNIQUE(IDENTIFIER)

CREATE SEQUENCE ITP.S_DEVICE"
Syntax error in SQL statement "CREATE TABLE ITP.T_DEVICE
( 
	PK_DEVICEID INT PRIMARY KEY NOT NULL, 
	IDENTIFIER INT NOT NULL, 
	ACTIVITY DATE NOT NULL, 
	LATITUDE DATE NOT NULL, 
	LONGITUDE DATE NOT NULL 
)  

ALTER[*] TABLE ITP.T_DEVICE ADD CONSTRAINT IDENTIFIER_UNIQUE UNIQUE(IDENTIFIER)

CREATE SEQUENCE ITP.S_DEVICE"; SQL statement:
CREATE TABLE ITP.T_DEVICE
( 
	PK_DEVICEID INT PRIMARY KEY NOT NULL, 
	IDENTIFIER INT NOT NULL, 
	ACTIVITY DATE NOT NULL, 
	LATITUDE DATE NOT NULL, 
	LONGITUDE DATE NOT NULL 
)  

ALTER TABLE ITP.T_DEVICE ADD CONSTRAINT IDENTIFIER_UNIQUE UNIQUE(IDENTIFIER)

CREATE SEQUENCE ITP.S_DEVICE [42000-170]
07-26 20:44:34 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Feld "00-B0-D0-86-BB-F7" nicht gefunden
Column "00-B0-D0-86-BB-F7" not found; SQL statement:
INSERT INTO ITP.T_DEVICE VALUES (NEXTVAL('ITP.S_DEVICE'), "00-B0-D0-86-BB-F7", CURRENT_DATE(), 49.456409, 11.078682) [42122-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:141)
	at org.h2.command.dml.Insert.prepare(Insert.java:231)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:414)
	at org.h2.engine.Session.prepareCommand(Session.java:363)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:164)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1311)
	at org.h2.server.web.WebApp.query(WebApp.java:1001)
	at org.h2.server.web.WebApp$1.next(WebApp.java:964)
	at org.h2.server.web.WebApp$1.next(WebApp.java:953)
	at org.h2.server.web.WebThread.process(WebThread.java:166)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
07-26 20:44:47 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Datenumwandlungsfehler beim Umwandeln von "'00-B0-D0-86-BB-F7' (T_DEVICE: IDENTIFIER INT NOT NULL)"
Data conversion error converting "'00-B0-D0-86-BB-F7' (T_DEVICE: IDENTIFIER INT NOT NULL)"; SQL statement:
INSERT INTO ITP.T_DEVICE VALUES (NEXTVAL('ITP.S_DEVICE'), '00-B0-D0-86-BB-F7', CURRENT_DATE(), 49.456409, 11.078682) -- (NEXTVAL('ITP.S_DEVICE'), '00-B0-D0-86-BB-F7', CURRENT_DATE(), 49.456409, 11.078682) [22018-170]
07-26 20:45:23 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Feld "00-B0-D0-86-BB-F7" nicht gefunden
Column "00-B0-D0-86-BB-F7" not found; SQL statement:
INSERT INTO ITP.T_DEVICE VALUES (NEXTVAL('ITP.S_DEVICE'), "00-B0-D0-86-BB-F7", CURRENT_DATE(), 49.456409, 11.078682) [42122-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:141)
	at org.h2.command.dml.Insert.prepare(Insert.java:231)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:414)
	at org.h2.engine.Session.prepareCommand(Session.java:363)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:164)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1311)
	at org.h2.server.web.WebApp.query(WebApp.java:1001)
	at org.h2.server.web.WebApp$1.next(WebApp.java:964)
	at org.h2.server.web.WebApp$1.next(WebApp.java:953)
	at org.h2.server.web.WebThread.process(WebThread.java:166)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
07-26 20:45:29 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Feld "Test" nicht gefunden
Column "Test" not found; SQL statement:
INSERT INTO ITP.T_DEVICE VALUES (NEXTVAL('ITP.S_DEVICE'), "Test", CURRENT_DATE(), 49.456409, 11.078682) [42122-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:141)
	at org.h2.command.dml.Insert.prepare(Insert.java:231)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:414)
	at org.h2.engine.Session.prepareCommand(Session.java:363)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:164)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1311)
	at org.h2.server.web.WebApp.query(WebApp.java:1001)
	at org.h2.server.web.WebApp$1.next(WebApp.java:964)
	at org.h2.server.web.WebApp$1.next(WebApp.java:953)
	at org.h2.server.web.WebThread.process(WebThread.java:166)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
07-26 20:46:05 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Feld "00-B0-D0-86-BB-F7" nicht gefunden
Column "00-B0-D0-86-BB-F7" not found; SQL statement:
INSERT INTO ITP.T_DEVICE (PK_DEVICEID, IDENTIFIER, ACTIVITY, LATITUDE, LONGITUDE) VALUES (NEXTVAL('ITP.S_DEVICE'), "00-B0-D0-86-BB-F7", CURRENT_DATE(), 49.456409, 11.078682) [42122-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:141)
	at org.h2.command.dml.Insert.prepare(Insert.java:231)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:414)
	at org.h2.engine.Session.prepareCommand(Session.java:363)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:164)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1311)
	at org.h2.server.web.WebApp.query(WebApp.java:1001)
	at org.h2.server.web.WebApp$1.next(WebApp.java:964)
	at org.h2.server.web.WebApp$1.next(WebApp.java:953)
	at org.h2.server.web.WebThread.process(WebThread.java:166)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
07-26 20:46:51 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Datenumwandlungsfehler beim Umwandeln von "'00-B0-D0-86-BB-F7' (T_DEVICE: IDENTIFIER INT NOT NULL)"
Data conversion error converting "'00-B0-D0-86-BB-F7' (T_DEVICE: IDENTIFIER INT NOT NULL)"; SQL statement:
INSERT INTO ITP.T_DEVICE (PK_DEVICEID, IDENTIFIER, ACTIVITY, LATITUDE, LONGITUDE) VALUES (NEXTVAL('ITP.S_DEVICE'), '00-B0-D0-86-BB-F7', CURRENT_DATE(), 49.456409, 11.078682) -- (NEXTVAL('ITP.S_DEVICE'), '00-B0-D0-86-BB-F7', CURRENT_DATE(), 49.456409, 11.078682) [22018-170]
07-26 20:48:37 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Datenumwandlungsfehler beim Umwandeln von "'00-B0-D0-86-BB-F7' (T_DEVICE: IDENTIFIER INT NOT NULL)"
Data conversion error converting "'00-B0-D0-86-BB-F7' (T_DEVICE: IDENTIFIER INT NOT NULL)"; SQL statement:
INSERT INTO ITP.T_DEVICE (PK_DEVICEID, IDENTIFIER, ACTIVITY, LATITUDE, LONGITUDE) VALUES (NEXTVAL('ITP.S_DEVICE'), '00-B0-D0-86-BB-F7', CURRENT_DATE(), 49.456409, 11.078682) -- (NEXTVAL('ITP.S_DEVICE'), '00-B0-D0-86-BB-F7', CURRENT_DATE(), 49.456409, 11.078682) [22018-170]
07-26 20:49:35 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Syntax Fehler in SQL Befehl "SELECT CURRENT_TIMESTAMP() FROM 1"; erwartet "identifier"
Syntax error in SQL statement "SELECT CURRENT_TIMESTAMP() FROM 1"; expected "identifier"; SQL statement:
SELECT CURRENT_TIMESTAMP() FROM 1 [42001-170]
07-26 20:49:48 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Datenumwandlungsfehler beim Umwandeln von "'00-B0-D0-86-BB-F7' (T_DEVICE: IDENTIFIER INT NOT NULL)"
Data conversion error converting "'00-B0-D0-86-BB-F7' (T_DEVICE: IDENTIFIER INT NOT NULL)"; SQL statement:
INSERT INTO ITP.T_DEVICE (PK_DEVICEID, IDENTIFIER, ACTIVITY, LATITUDE, LONGITUDE) VALUES (NEXTVAL('ITP.S_DEVICE'), '00-B0-D0-86-BB-F7', CURRENT_TIMESTAMP(), 49.456409, 11.078682) -- (NEXTVAL('ITP.S_DEVICE'), '00-B0-D0-86-BB-F7', CURRENT_TIMESTAMP(), 49.456409, 11.078682) [22018-170]
07-27 19:53:46 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Das Objekt wurde bereits geschlossen
The object is already closed [90007-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcResultSet.checkClosed(JdbcResultSet.java:2945)
	at org.h2.jdbc.JdbcResultSet.next(JdbcResultSet.java:123)
	at com.th.nuernberg.itp.webservice.DeviceResource.list(DeviceResource.java:68)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.sun.jersey.spi.container.JavaMethodInvokerFactory$1.invoke(JavaMethodInvokerFactory.java:60)
	at com.sun.jersey.server.impl.model.method.dispatch.AbstractResourceMethodDispatchProvider$TypeOutInvoker._dispatch(AbstractResourceMethodDispatchProvider.java:185)
	at com.sun.jersey.server.impl.model.method.dispatch.ResourceJavaMethodDispatcher.dispatch(ResourceJavaMethodDispatcher.java:75)
	at com.sun.jersey.server.impl.uri.rules.HttpMethodRule.accept(HttpMethodRule.java:288)
	at com.sun.jersey.server.impl.uri.rules.RightHandPathRule.accept(RightHandPathRule.java:147)
	at com.sun.jersey.server.impl.uri.rules.ResourceClassRule.accept(ResourceClassRule.java:108)
	at com.sun.jersey.server.impl.uri.rules.RightHandPathRule.accept(RightHandPathRule.java:147)
	at com.sun.jersey.server.impl.uri.rules.RootResourceClassesRule.accept(RootResourceClassesRule.java:84)
	at com.sun.jersey.server.impl.application.WebApplicationImpl._handleRequest(WebApplicationImpl.java:1469)
	at com.sun.jersey.server.impl.application.WebApplicationImpl._handleRequest(WebApplicationImpl.java:1400)
	at com.sun.jersey.server.impl.application.WebApplicationImpl.handleRequest(WebApplicationImpl.java:1349)
	at com.sun.jersey.server.impl.application.WebApplicationImpl.handleRequest(WebApplicationImpl.java:1339)
	at com.sun.jersey.server.impl.container.httpserver.HttpHandlerContainer.handle(HttpHandlerContainer.java:191)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.AuthFilter.doFilter(Unknown Source)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.ServerImpl$Exchange$LinkHandler.handle(Unknown Source)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.ServerImpl$Exchange.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
07-27 19:54:45 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Das Objekt wurde bereits geschlossen
The object is already closed [90007-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcResultSet.checkClosed(JdbcResultSet.java:2945)
	at org.h2.jdbc.JdbcResultSet.next(JdbcResultSet.java:123)
	at com.th.nuernberg.itp.webservice.DeviceResource.list(DeviceResource.java:68)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.sun.jersey.spi.container.JavaMethodInvokerFactory$1.invoke(JavaMethodInvokerFactory.java:60)
	at com.sun.jersey.server.impl.model.method.dispatch.AbstractResourceMethodDispatchProvider$TypeOutInvoker._dispatch(AbstractResourceMethodDispatchProvider.java:185)
	at com.sun.jersey.server.impl.model.method.dispatch.ResourceJavaMethodDispatcher.dispatch(ResourceJavaMethodDispatcher.java:75)
	at com.sun.jersey.server.impl.uri.rules.HttpMethodRule.accept(HttpMethodRule.java:288)
	at com.sun.jersey.server.impl.uri.rules.RightHandPathRule.accept(RightHandPathRule.java:147)
	at com.sun.jersey.server.impl.uri.rules.ResourceClassRule.accept(ResourceClassRule.java:108)
	at com.sun.jersey.server.impl.uri.rules.RightHandPathRule.accept(RightHandPathRule.java:147)
	at com.sun.jersey.server.impl.uri.rules.RootResourceClassesRule.accept(RootResourceClassesRule.java:84)
	at com.sun.jersey.server.impl.application.WebApplicationImpl._handleRequest(WebApplicationImpl.java:1469)
	at com.sun.jersey.server.impl.application.WebApplicationImpl._handleRequest(WebApplicationImpl.java:1400)
	at com.sun.jersey.server.impl.application.WebApplicationImpl.handleRequest(WebApplicationImpl.java:1349)
	at com.sun.jersey.server.impl.application.WebApplicationImpl.handleRequest(WebApplicationImpl.java:1339)
	at com.sun.jersey.server.impl.container.httpserver.HttpHandlerContainer.handle(HttpHandlerContainer.java:191)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.AuthFilter.doFilter(Unknown Source)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.ServerImpl$Exchange$LinkHandler.handle(Unknown Source)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.ServerImpl$Exchange.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
07-27 19:56:51 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Das Objekt wurde bereits geschlossen
The object is already closed [90007-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcResultSet.checkClosed(JdbcResultSet.java:2945)
	at org.h2.jdbc.JdbcResultSet.next(JdbcResultSet.java:123)
	at com.th.nuernberg.itp.webservice.DeviceResource.list(DeviceResource.java:75)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.sun.jersey.spi.container.JavaMethodInvokerFactory$1.invoke(JavaMethodInvokerFactory.java:60)
	at com.sun.jersey.server.impl.model.method.dispatch.AbstractResourceMethodDispatchProvider$TypeOutInvoker._dispatch(AbstractResourceMethodDispatchProvider.java:185)
	at com.sun.jersey.server.impl.model.method.dispatch.ResourceJavaMethodDispatcher.dispatch(ResourceJavaMethodDispatcher.java:75)
	at com.sun.jersey.server.impl.uri.rules.HttpMethodRule.accept(HttpMethodRule.java:288)
	at com.sun.jersey.server.impl.uri.rules.RightHandPathRule.accept(RightHandPathRule.java:147)
	at com.sun.jersey.server.impl.uri.rules.ResourceClassRule.accept(ResourceClassRule.java:108)
	at com.sun.jersey.server.impl.uri.rules.RightHandPathRule.accept(RightHandPathRule.java:147)
	at com.sun.jersey.server.impl.uri.rules.RootResourceClassesRule.accept(RootResourceClassesRule.java:84)
	at com.sun.jersey.server.impl.application.WebApplicationImpl._handleRequest(WebApplicationImpl.java:1469)
	at com.sun.jersey.server.impl.application.WebApplicationImpl._handleRequest(WebApplicationImpl.java:1400)
	at com.sun.jersey.server.impl.application.WebApplicationImpl.handleRequest(WebApplicationImpl.java:1349)
	at com.sun.jersey.server.impl.application.WebApplicationImpl.handleRequest(WebApplicationImpl.java:1339)
	at com.sun.jersey.server.impl.container.httpserver.HttpHandlerContainer.handle(HttpHandlerContainer.java:191)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.AuthFilter.doFilter(Unknown Source)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.ServerImpl$Exchange$LinkHandler.handle(Unknown Source)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.ServerImpl$Exchange.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
07-27 19:58:34 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Unerlaubter Wert "0" für Parameter "columnIndex"
Invalid value "0" for parameter "columnIndex" [90008-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.getInvalidValueException(DbException.java:215)
	at org.h2.jdbc.JdbcResultSet.checkColumnIndex(JdbcResultSet.java:2934)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:2962)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:275)
	at com.th.nuernberg.itp.webservice.DeviceResource.list(DeviceResource.java:76)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.sun.jersey.spi.container.JavaMethodInvokerFactory$1.invoke(JavaMethodInvokerFactory.java:60)
	at com.sun.jersey.server.impl.model.method.dispatch.AbstractResourceMethodDispatchProvider$TypeOutInvoker._dispatch(AbstractResourceMethodDispatchProvider.java:185)
	at com.sun.jersey.server.impl.model.method.dispatch.ResourceJavaMethodDispatcher.dispatch(ResourceJavaMethodDispatcher.java:75)
	at com.sun.jersey.server.impl.uri.rules.HttpMethodRule.accept(HttpMethodRule.java:288)
	at com.sun.jersey.server.impl.uri.rules.RightHandPathRule.accept(RightHandPathRule.java:147)
	at com.sun.jersey.server.impl.uri.rules.ResourceClassRule.accept(ResourceClassRule.java:108)
	at com.sun.jersey.server.impl.uri.rules.RightHandPathRule.accept(RightHandPathRule.java:147)
	at com.sun.jersey.server.impl.uri.rules.RootResourceClassesRule.accept(RootResourceClassesRule.java:84)
	at com.sun.jersey.server.impl.application.WebApplicationImpl._handleRequest(WebApplicationImpl.java:1469)
	at com.sun.jersey.server.impl.application.WebApplicationImpl._handleRequest(WebApplicationImpl.java:1400)
	at com.sun.jersey.server.impl.application.WebApplicationImpl.handleRequest(WebApplicationImpl.java:1349)
	at com.sun.jersey.server.impl.application.WebApplicationImpl.handleRequest(WebApplicationImpl.java:1339)
	at com.sun.jersey.server.impl.container.httpserver.HttpHandlerContainer.handle(HttpHandlerContainer.java:191)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.AuthFilter.doFilter(Unknown Source)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.ServerImpl$Exchange$LinkHandler.handle(Unknown Source)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.ServerImpl$Exchange.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
07-27 19:58:36 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Unerlaubter Wert "0" für Parameter "columnIndex"
Invalid value "0" for parameter "columnIndex" [90008-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.getInvalidValueException(DbException.java:215)
	at org.h2.jdbc.JdbcResultSet.checkColumnIndex(JdbcResultSet.java:2934)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:2962)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:275)
	at com.th.nuernberg.itp.webservice.DeviceResource.list(DeviceResource.java:76)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.sun.jersey.spi.container.JavaMethodInvokerFactory$1.invoke(JavaMethodInvokerFactory.java:60)
	at com.sun.jersey.server.impl.model.method.dispatch.AbstractResourceMethodDispatchProvider$TypeOutInvoker._dispatch(AbstractResourceMethodDispatchProvider.java:185)
	at com.sun.jersey.server.impl.model.method.dispatch.ResourceJavaMethodDispatcher.dispatch(ResourceJavaMethodDispatcher.java:75)
	at com.sun.jersey.server.impl.uri.rules.HttpMethodRule.accept(HttpMethodRule.java:288)
	at com.sun.jersey.server.impl.uri.rules.RightHandPathRule.accept(RightHandPathRule.java:147)
	at com.sun.jersey.server.impl.uri.rules.ResourceClassRule.accept(ResourceClassRule.java:108)
	at com.sun.jersey.server.impl.uri.rules.RightHandPathRule.accept(RightHandPathRule.java:147)
	at com.sun.jersey.server.impl.uri.rules.RootResourceClassesRule.accept(RootResourceClassesRule.java:84)
	at com.sun.jersey.server.impl.application.WebApplicationImpl._handleRequest(WebApplicationImpl.java:1469)
	at com.sun.jersey.server.impl.application.WebApplicationImpl._handleRequest(WebApplicationImpl.java:1400)
	at com.sun.jersey.server.impl.application.WebApplicationImpl.handleRequest(WebApplicationImpl.java:1349)
	at com.sun.jersey.server.impl.application.WebApplicationImpl.handleRequest(WebApplicationImpl.java:1339)
	at com.sun.jersey.server.impl.container.httpserver.HttpHandlerContainer.handle(HttpHandlerContainer.java:191)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.AuthFilter.doFilter(Unknown Source)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.ServerImpl$Exchange$LinkHandler.handle(Unknown Source)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.ServerImpl$Exchange.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
07-28 13:38:27 jdbc[5]: null
org.h2.message.DbException: Die Datenbank-Verbindung wurde nicht explizit geschlossen (jetzt in der Müllabfuhr)
The connection was not closed by the application and is garbage collected [90018-170]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:175)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.th.nuernberg.itp.webservice.Database.Initalize(Database.java:19)
	at com.th.nuernberg.itp.webservice.BaseResource.createDatabaseInstance(BaseResource.java:36)
	at com.th.nuernberg.itp.webservice.DeviceResource.<init>(DeviceResource.java:21)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.sun.jersey.server.spi.component.ResourceComponentConstructor._construct(ResourceComponentConstructor.java:191)
	at com.sun.jersey.server.spi.component.ResourceComponentConstructor.construct(ResourceComponentConstructor.java:179)
	at com.sun.jersey.server.impl.resource.PerRequestFactory$PerRequest._getInstance(PerRequestFactory.java:182)
	at com.sun.jersey.server.impl.resource.PerRequestFactory$AbstractPerRequest.getInstance(PerRequestFactory.java:144)
	at com.sun.jersey.server.impl.application.WebApplicationContext.getResource(WebApplicationContext.java:238)
	at com.sun.jersey.server.impl.uri.rules.ResourceClassRule.accept(ResourceClassRule.java:83)
	at com.sun.jersey.server.impl.uri.rules.RightHandPathRule.accept(RightHandPathRule.java:147)
	at com.sun.jersey.server.impl.uri.rules.RootResourceClassesRule.accept(RootResourceClassesRule.java:84)
	at com.sun.jersey.server.impl.application.WebApplicationImpl._handleRequest(WebApplicationImpl.java:1469)
	at com.sun.jersey.server.impl.application.WebApplicationImpl._handleRequest(WebApplicationImpl.java:1400)
	at com.sun.jersey.server.impl.application.WebApplicationImpl.handleRequest(WebApplicationImpl.java:1349)
	at com.sun.jersey.server.impl.application.WebApplicationImpl.handleRequest(WebApplicationImpl.java:1339)
	at com.sun.jersey.server.impl.container.httpserver.HttpHandlerContainer.handle(HttpHandlerContainer.java:191)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.AuthFilter.doFilter(Unknown Source)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.ServerImpl$Exchange$LinkHandler.handle(Unknown Source)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.ServerImpl$Exchange.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: Die Datenbank-Verbindung wurde nicht explizit geschlossen (jetzt in der Müllabfuhr)
The connection was not closed by the application and is garbage collected [90018-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 38 more
07-28 13:38:27 jdbc[5]: null
org.h2.message.DbException: Die Datenbank-Verbindung wurde nicht explizit geschlossen (jetzt in der Müllabfuhr)
The connection was not closed by the application and is garbage collected [90018-170]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:175)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.th.nuernberg.itp.webservice.Database.Initalize(Database.java:19)
	at com.th.nuernberg.itp.webservice.BaseResource.createDatabaseInstance(BaseResource.java:36)
	at com.th.nuernberg.itp.webservice.DeviceResource.<init>(DeviceResource.java:21)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.sun.jersey.server.spi.component.ResourceComponentConstructor._construct(ResourceComponentConstructor.java:191)
	at com.sun.jersey.server.spi.component.ResourceComponentConstructor.construct(ResourceComponentConstructor.java:179)
	at com.sun.jersey.server.impl.resource.PerRequestFactory$PerRequest._getInstance(PerRequestFactory.java:182)
	at com.sun.jersey.server.impl.resource.PerRequestFactory$AbstractPerRequest.getInstance(PerRequestFactory.java:144)
	at com.sun.jersey.server.impl.application.WebApplicationContext.getResource(WebApplicationContext.java:238)
	at com.sun.jersey.server.impl.uri.rules.ResourceClassRule.accept(ResourceClassRule.java:83)
	at com.sun.jersey.server.impl.uri.rules.RightHandPathRule.accept(RightHandPathRule.java:147)
	at com.sun.jersey.server.impl.uri.rules.RootResourceClassesRule.accept(RootResourceClassesRule.java:84)
	at com.sun.jersey.server.impl.application.WebApplicationImpl._handleRequest(WebApplicationImpl.java:1469)
	at com.sun.jersey.server.impl.application.WebApplicationImpl._handleRequest(WebApplicationImpl.java:1400)
	at com.sun.jersey.server.impl.application.WebApplicationImpl.handleRequest(WebApplicationImpl.java:1349)
	at com.sun.jersey.server.impl.application.WebApplicationImpl.handleRequest(WebApplicationImpl.java:1339)
	at com.sun.jersey.server.impl.container.httpserver.HttpHandlerContainer.handle(HttpHandlerContainer.java:191)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.AuthFilter.doFilter(Unknown Source)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.ServerImpl$Exchange$LinkHandler.handle(Unknown Source)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.ServerImpl$Exchange.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: Die Datenbank-Verbindung wurde nicht explizit geschlossen (jetzt in der Müllabfuhr)
The connection was not closed by the application and is garbage collected [90018-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 38 more
07-28 13:42:11 jdbc[5]: null
org.h2.message.DbException: Die Datenbank-Verbindung wurde nicht explizit geschlossen (jetzt in der Müllabfuhr)
The connection was not closed by the application and is garbage collected [90018-170]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:175)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.th.nuernberg.itp.webservice.Database.Initalize(Database.java:19)
	at com.th.nuernberg.itp.webservice.BaseResource.createDatabaseInstance(BaseResource.java:36)
	at com.th.nuernberg.itp.webservice.DeviceResource.<init>(DeviceResource.java:21)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.sun.jersey.server.spi.component.ResourceComponentConstructor._construct(ResourceComponentConstructor.java:191)
	at com.sun.jersey.server.spi.component.ResourceComponentConstructor.construct(ResourceComponentConstructor.java:179)
	at com.sun.jersey.server.impl.resource.PerRequestFactory$PerRequest._getInstance(PerRequestFactory.java:182)
	at com.sun.jersey.server.impl.resource.PerRequestFactory$AbstractPerRequest.getInstance(PerRequestFactory.java:144)
	at com.sun.jersey.server.impl.application.WebApplicationContext.getResource(WebApplicationContext.java:238)
	at com.sun.jersey.server.impl.uri.rules.ResourceClassRule.accept(ResourceClassRule.java:83)
	at com.sun.jersey.server.impl.uri.rules.RightHandPathRule.accept(RightHandPathRule.java:147)
	at com.sun.jersey.server.impl.uri.rules.RootResourceClassesRule.accept(RootResourceClassesRule.java:84)
	at com.sun.jersey.server.impl.application.WebApplicationImpl._handleRequest(WebApplicationImpl.java:1469)
	at com.sun.jersey.server.impl.application.WebApplicationImpl._handleRequest(WebApplicationImpl.java:1400)
	at com.sun.jersey.server.impl.application.WebApplicationImpl.handleRequest(WebApplicationImpl.java:1349)
	at com.sun.jersey.server.impl.application.WebApplicationImpl.handleRequest(WebApplicationImpl.java:1339)
	at com.sun.jersey.server.impl.container.httpserver.HttpHandlerContainer.handle(HttpHandlerContainer.java:191)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.AuthFilter.doFilter(Unknown Source)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.ServerImpl$Exchange$LinkHandler.handle(Unknown Source)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.ServerImpl$Exchange.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: Die Datenbank-Verbindung wurde nicht explizit geschlossen (jetzt in der Müllabfuhr)
The connection was not closed by the application and is garbage collected [90018-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 38 more
07-28 13:42:11 jdbc[5]: null
org.h2.message.DbException: Die Datenbank-Verbindung wurde nicht explizit geschlossen (jetzt in der Müllabfuhr)
The connection was not closed by the application and is garbage collected [90018-170]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:175)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.th.nuernberg.itp.webservice.Database.Initalize(Database.java:19)
	at com.th.nuernberg.itp.webservice.BaseResource.createDatabaseInstance(BaseResource.java:36)
	at com.th.nuernberg.itp.webservice.DeviceResource.<init>(DeviceResource.java:21)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.sun.jersey.server.spi.component.ResourceComponentConstructor._construct(ResourceComponentConstructor.java:191)
	at com.sun.jersey.server.spi.component.ResourceComponentConstructor.construct(ResourceComponentConstructor.java:179)
	at com.sun.jersey.server.impl.resource.PerRequestFactory$PerRequest._getInstance(PerRequestFactory.java:182)
	at com.sun.jersey.server.impl.resource.PerRequestFactory$AbstractPerRequest.getInstance(PerRequestFactory.java:144)
	at com.sun.jersey.server.impl.application.WebApplicationContext.getResource(WebApplicationContext.java:238)
	at com.sun.jersey.server.impl.uri.rules.ResourceClassRule.accept(ResourceClassRule.java:83)
	at com.sun.jersey.server.impl.uri.rules.RightHandPathRule.accept(RightHandPathRule.java:147)
	at com.sun.jersey.server.impl.uri.rules.RootResourceClassesRule.accept(RootResourceClassesRule.java:84)
	at com.sun.jersey.server.impl.application.WebApplicationImpl._handleRequest(WebApplicationImpl.java:1469)
	at com.sun.jersey.server.impl.application.WebApplicationImpl._handleRequest(WebApplicationImpl.java:1400)
	at com.sun.jersey.server.impl.application.WebApplicationImpl.handleRequest(WebApplicationImpl.java:1349)
	at com.sun.jersey.server.impl.application.WebApplicationImpl.handleRequest(WebApplicationImpl.java:1339)
	at com.sun.jersey.server.impl.container.httpserver.HttpHandlerContainer.handle(HttpHandlerContainer.java:191)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.AuthFilter.doFilter(Unknown Source)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.ServerImpl$Exchange$LinkHandler.handle(Unknown Source)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.ServerImpl$Exchange.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: Die Datenbank-Verbindung wurde nicht explizit geschlossen (jetzt in der Müllabfuhr)
The connection was not closed by the application and is garbage collected [90018-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 38 more
07-28 15:01:43 jdbc[4]: null
org.h2.message.DbException: Die Datenbank-Verbindung wurde nicht explizit geschlossen (jetzt in der Müllabfuhr)
The connection was not closed by the application and is garbage collected [90018-170]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:175)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.th.nuernberg.itp.webservice.Database.Initalize(Database.java:19)
	at com.th.nuernberg.itp.webservice.BaseResource.createDatabaseInstance(BaseResource.java:36)
	at com.th.nuernberg.itp.webservice.DeviceResource.<init>(DeviceResource.java:21)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.sun.jersey.server.spi.component.ResourceComponentConstructor._construct(ResourceComponentConstructor.java:191)
	at com.sun.jersey.server.spi.component.ResourceComponentConstructor.construct(ResourceComponentConstructor.java:179)
	at com.sun.jersey.server.impl.resource.PerRequestFactory$PerRequest._getInstance(PerRequestFactory.java:182)
	at com.sun.jersey.server.impl.resource.PerRequestFactory$AbstractPerRequest.getInstance(PerRequestFactory.java:144)
	at com.sun.jersey.server.impl.application.WebApplicationContext.getResource(WebApplicationContext.java:238)
	at com.sun.jersey.server.impl.uri.rules.ResourceClassRule.accept(ResourceClassRule.java:83)
	at com.sun.jersey.server.impl.uri.rules.RightHandPathRule.accept(RightHandPathRule.java:147)
	at com.sun.jersey.server.impl.uri.rules.RootResourceClassesRule.accept(RootResourceClassesRule.java:84)
	at com.sun.jersey.server.impl.application.WebApplicationImpl._handleRequest(WebApplicationImpl.java:1469)
	at com.sun.jersey.server.impl.application.WebApplicationImpl._handleRequest(WebApplicationImpl.java:1400)
	at com.sun.jersey.server.impl.application.WebApplicationImpl.handleRequest(WebApplicationImpl.java:1349)
	at com.sun.jersey.server.impl.application.WebApplicationImpl.handleRequest(WebApplicationImpl.java:1339)
	at com.sun.jersey.server.impl.container.httpserver.HttpHandlerContainer.handle(HttpHandlerContainer.java:191)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.AuthFilter.doFilter(Unknown Source)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.ServerImpl$Exchange$LinkHandler.handle(Unknown Source)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.ServerImpl$Exchange.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: Die Datenbank-Verbindung wurde nicht explizit geschlossen (jetzt in der Müllabfuhr)
The connection was not closed by the application and is garbage collected [90018-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 38 more
07-28 16:14:05 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Datenumwandlungsfehler beim Umwandeln von "2013-07-27 19:02:15.776"
Data conversion error converting "2013-07-27 19:02:15.776" [22018-170]
07-28 16:14:45 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Datenumwandlungsfehler beim Umwandeln von "2013-07-27 19:02:15.776"
Data conversion error converting "2013-07-27 19:02:15.776" [22018-170]
10-04 19:55:31 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Feld "FK_DEVICEID" nicht gefunden
Column "FK_DEVICEID" not found; SQL statement:
ALTER TABLE ITP.T_DEVICE ADD FOREIGN KEY(FK_DEVICEID) REFERENCES ITP.T_DEVICE(FK_DEVICEID) [42122-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.table.Table.getColumn(Table.java:605)
	at org.h2.table.IndexColumn.mapColumns(IndexColumn.java:75)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:181)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:68)
	at org.h2.command.CommandContainer.update(CommandContainer.java:75)
	at org.h2.command.Command.executeUpdate(Command.java:230)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:177)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1311)
	at org.h2.server.web.WebApp.query(WebApp.java:1001)
	at org.h2.server.web.WebApp$1.next(WebApp.java:964)
	at org.h2.server.web.WebApp$1.next(WebApp.java:953)
	at org.h2.server.web.WebThread.process(WebThread.java:166)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
10-04 19:55:43 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Feld "FK_DEVICEID" nicht gefunden
Column "FK_DEVICEID" not found; SQL statement:
ALTER TABLE ITP.T_DEVICE ADD FOREIGN KEY(FK_DEVICEID) REFERENCES ITP.T_DEVICE(PK_DEVICEID) [42122-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.table.Table.getColumn(Table.java:605)
	at org.h2.table.IndexColumn.mapColumns(IndexColumn.java:75)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:181)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:68)
	at org.h2.command.CommandContainer.update(CommandContainer.java:75)
	at org.h2.command.Command.executeUpdate(Command.java:230)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:177)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1311)
	at org.h2.server.web.WebApp.query(WebApp.java:1001)
	at org.h2.server.web.WebApp$1.next(WebApp.java:964)
	at org.h2.server.web.WebApp$1.next(WebApp.java:953)
	at org.h2.server.web.WebThread.process(WebThread.java:166)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
10-04 19:55:58 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Feld "FK_DEVICEID" nicht gefunden
Column "FK_DEVICEID" not found; SQL statement:
ALTER TABLE ITP.T_DEVICE ADD FOREIGN KEY(FK_DEVICEID) REFERENCES ITP.T_DEVICE(PK_DEVICEID) [42122-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.table.Table.getColumn(Table.java:605)
	at org.h2.table.IndexColumn.mapColumns(IndexColumn.java:75)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:181)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:68)
	at org.h2.command.CommandContainer.update(CommandContainer.java:75)
	at org.h2.command.Command.executeUpdate(Command.java:230)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:177)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1311)
	at org.h2.server.web.WebApp.query(WebApp.java:1001)
	at org.h2.server.web.WebApp$1.next(WebApp.java:964)
	at org.h2.server.web.WebApp$1.next(WebApp.java:953)
	at org.h2.server.web.WebThread.process(WebThread.java:166)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
10-04 21:58:43 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Kann "TIMESTAMP" "20131004_215843" nicht umwandeln
Cannot parse "TIMESTAMP" constant "20131004_215843"; SQL statement:
INSERT INTO ITP.T_DEVICE (PK_DEVICEID, IDENTIFIER, ACTIVITY, LATITUDE, LONGITUDE) VALUES (NEXTVAL('ITP.S_DEVICE'), '00b0d086bbf0', '20131004_215843', 11.078682, 49.456409) -- (NEXTVAL('ITP.S_DEVICE'), '00b0d086bbf0', '20131004_215843', 11.078682, 49.456409) [22007-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:158)
	at org.h2.value.ValueTimestamp.parse(ValueTimestamp.java:87)
	at org.h2.value.Value.convertTo(Value.java:819)
	at org.h2.table.Column.convert(Column.java:143)
	at org.h2.command.dml.Insert.insertRows(Insert.java:112)
	at org.h2.command.dml.Insert.update(Insert.java:84)
	at org.h2.command.CommandContainer.update(CommandContainer.java:75)
	at org.h2.command.Command.executeUpdate(Command.java:230)
	at org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:125)
	at org.h2.jdbc.JdbcStatement.executeUpdate(JdbcStatement.java:110)
	at com.th.nuernberg.itp.webservice.Database.execute(Database.java:28)
	at com.th.nuernberg.itp.webservice.types.DeviceRepository.persist(DeviceRepository.java:23)
	at com.th.nuernberg.itp.webservice.DeviceResource.register(DeviceResource.java:46)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.sun.jersey.spi.container.JavaMethodInvokerFactory$1.invoke(JavaMethodInvokerFactory.java:60)
	at com.sun.jersey.server.impl.model.method.dispatch.AbstractResourceMethodDispatchProvider$TypeOutInvoker._dispatch(AbstractResourceMethodDispatchProvider.java:185)
	at com.sun.jersey.server.impl.model.method.dispatch.ResourceJavaMethodDispatcher.dispatch(ResourceJavaMethodDispatcher.java:75)
	at com.sun.jersey.server.impl.uri.rules.HttpMethodRule.accept(HttpMethodRule.java:288)
	at com.sun.jersey.server.impl.uri.rules.RightHandPathRule.accept(RightHandPathRule.java:147)
	at com.sun.jersey.server.impl.uri.rules.ResourceClassRule.accept(ResourceClassRule.java:108)
	at com.sun.jersey.server.impl.uri.rules.RightHandPathRule.accept(RightHandPathRule.java:147)
	at com.sun.jersey.server.impl.uri.rules.RootResourceClassesRule.accept(RootResourceClassesRule.java:84)
	at com.sun.jersey.server.impl.application.WebApplicationImpl._handleRequest(WebApplicationImpl.java:1469)
	at com.sun.jersey.server.impl.application.WebApplicationImpl._handleRequest(WebApplicationImpl.java:1400)
	at com.sun.jersey.server.impl.application.WebApplicationImpl.handleRequest(WebApplicationImpl.java:1349)
	at com.sun.jersey.server.impl.application.WebApplicationImpl.handleRequest(WebApplicationImpl.java:1339)
	at com.sun.jersey.server.impl.container.httpserver.HttpHandlerContainer.handle(HttpHandlerContainer.java:191)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.AuthFilter.doFilter(Unknown Source)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.ServerImpl$Exchange$LinkHandler.handle(Unknown Source)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.ServerImpl$Exchange.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.IllegalArgumentException: 20131004_215843
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:270)
	at org.h2.value.ValueTimestamp.parseTry(ValueTimestamp.java:105)
	at org.h2.value.ValueTimestamp.parse(ValueTimestamp.java:85)
	... 37 more
10-04 21:58:56 jdbc[8]: exception
org.h2.jdbc.JdbcSQLException: Kann "TIMESTAMP" "20131004_215856" nicht umwandeln
Cannot parse "TIMESTAMP" constant "20131004_215856"; SQL statement:
INSERT INTO ITP.T_DEVICE (PK_DEVICEID, IDENTIFIER, ACTIVITY, LATITUDE, LONGITUDE) VALUES (NEXTVAL('ITP.S_DEVICE'), '00b0d086bbf0', '20131004_215856', 11.078682, 49.456409) -- (NEXTVAL('ITP.S_DEVICE'), '00b0d086bbf0', '20131004_215856', 11.078682, 49.456409) [22007-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:158)
	at org.h2.value.ValueTimestamp.parse(ValueTimestamp.java:87)
	at org.h2.value.Value.convertTo(Value.java:819)
	at org.h2.table.Column.convert(Column.java:143)
	at org.h2.command.dml.Insert.insertRows(Insert.java:112)
	at org.h2.command.dml.Insert.update(Insert.java:84)
	at org.h2.command.CommandContainer.update(CommandContainer.java:75)
	at org.h2.command.Command.executeUpdate(Command.java:230)
	at org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:125)
	at org.h2.jdbc.JdbcStatement.executeUpdate(JdbcStatement.java:110)
	at com.th.nuernberg.itp.webservice.Database.execute(Database.java:28)
	at com.th.nuernberg.itp.webservice.types.DeviceRepository.persist(DeviceRepository.java:23)
	at com.th.nuernberg.itp.webservice.DeviceResource.register(DeviceResource.java:46)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.sun.jersey.spi.container.JavaMethodInvokerFactory$1.invoke(JavaMethodInvokerFactory.java:60)
	at com.sun.jersey.server.impl.model.method.dispatch.AbstractResourceMethodDispatchProvider$TypeOutInvoker._dispatch(AbstractResourceMethodDispatchProvider.java:185)
	at com.sun.jersey.server.impl.model.method.dispatch.ResourceJavaMethodDispatcher.dispatch(ResourceJavaMethodDispatcher.java:75)
	at com.sun.jersey.server.impl.uri.rules.HttpMethodRule.accept(HttpMethodRule.java:288)
	at com.sun.jersey.server.impl.uri.rules.RightHandPathRule.accept(RightHandPathRule.java:147)
	at com.sun.jersey.server.impl.uri.rules.ResourceClassRule.accept(ResourceClassRule.java:108)
	at com.sun.jersey.server.impl.uri.rules.RightHandPathRule.accept(RightHandPathRule.java:147)
	at com.sun.jersey.server.impl.uri.rules.RootResourceClassesRule.accept(RootResourceClassesRule.java:84)
	at com.sun.jersey.server.impl.application.WebApplicationImpl._handleRequest(WebApplicationImpl.java:1469)
	at com.sun.jersey.server.impl.application.WebApplicationImpl._handleRequest(WebApplicationImpl.java:1400)
	at com.sun.jersey.server.impl.application.WebApplicationImpl.handleRequest(WebApplicationImpl.java:1349)
	at com.sun.jersey.server.impl.application.WebApplicationImpl.handleRequest(WebApplicationImpl.java:1339)
	at com.sun.jersey.server.impl.container.httpserver.HttpHandlerContainer.handle(HttpHandlerContainer.java:191)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.AuthFilter.doFilter(Unknown Source)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.ServerImpl$Exchange$LinkHandler.handle(Unknown Source)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.ServerImpl$Exchange.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.IllegalArgumentException: 20131004_215856
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:270)
	at org.h2.value.ValueTimestamp.parseTry(ValueTimestamp.java:105)
	at org.h2.value.ValueTimestamp.parse(ValueTimestamp.java:85)
	... 37 more
10-05 11:35:01 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Bedingung "LEVELCHECK" besteht bereits
Constraint "LEVELCHECK" already exists; SQL statement:
ALTER TABLE  ITP.T_NOTIFICATION 
ADD CONSTRAINT LEVELCHECK
CHECK  (level < 4) [90045-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:89)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:68)
	at org.h2.command.CommandContainer.update(CommandContainer.java:75)
	at org.h2.command.Command.executeUpdate(Command.java:230)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:177)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1311)
	at org.h2.server.web.WebApp.query(WebApp.java:1001)
	at org.h2.server.web.WebApp$1.next(WebApp.java:964)
	at org.h2.server.web.WebApp$1.next(WebApp.java:953)
	at org.h2.server.web.WebThread.process(WebThread.java:166)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
10-31 18:39:24 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Tabelle "TEST" nicht gefunden
Table "TEST" not found; SQL statement:
ALTER TABLE TEST ALTER COLUMN IDENTIFIER VARCHAR(55) [42102-170]
10-31 18:39:49 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Syntax Fehler in SQL Befehl "DESC[*] ITP.T_DEVICE"; erwartet "DELETE, DROP, DECLARE, DEALLOCATE, {"
Syntax error in SQL statement "DESC[*] ITP.T_DEVICE"; expected "DELETE, DROP, DECLARE, DEALLOCATE, {"; SQL statement:
DESC ITP.T_DEVICE [42001-170]
10-31 18:40:19 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Wert zu gross / lang für Feld "IDENTIFIER VARCHAR(55) NOT NULL": "'APA91bGn8BTixyaOUTo_296LeE6xXL3S8PW1oDr0wyGhp8cKTadibajflRul52P-kTclJiSOROa8a72L_moznYk-5ihaGnyEylF06eqX9_oDIFjJmtgxYvMZG4kiwBd... (183)"
Value too long for column "IDENTIFIER VARCHAR(55) NOT NULL": "'APA91bGn8BTixyaOUTo_296LeE6xXL3S8PW1oDr0wyGhp8cKTadibajflRul52P-kTclJiSOROa8a72L_moznYk-5ihaGnyEylF06eqX9_oDIFjJmtgxYvMZG4kiwBd... (183)"; SQL statement:
UPDATE "ITP"."T_DEVICE" SET "PK_DEVICEID"=? ,"IDENTIFIER"=? ,"ACTIVITY"=? ,"LATITUDE"=? ,"LONGITUDE"=?  WHERE "PK_DEVICEID"=? [22001-170]
10-31 18:40:19 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Wert zu gross / lang für Feld "IDENTIFIER VARCHAR(55) NOT NULL": "'APA91bGn8BTixyaOUTo_296LeE6xXL3S8PW1oDr0wyGhp8cKTadibajflRul52P-kTclJiSOROa8a72L_moznYk-5ihaGnyEylF06eqX9_oDIFjJmtgxYvMZG4kiwBd... (183)"
Value too long for column "IDENTIFIER VARCHAR(55) NOT NULL": "'APA91bGn8BTixyaOUTo_296LeE6xXL3S8PW1oDr0wyGhp8cKTadibajflRul52P-kTclJiSOROa8a72L_moznYk-5ihaGnyEylF06eqX9_oDIFjJmtgxYvMZG4kiwBd... (183)"; SQL statement:
UPDATE "ITP"."T_DEVICE" SET "PK_DEVICEID"=? ,"IDENTIFIER"=? ,"ACTIVITY"=? ,"LATITUDE"=? ,"LONGITUDE"=?  WHERE "PK_DEVICEID"=? [22001-170]
10-31 19:48:41 jdbc[5]: null
org.h2.message.DbException: Die Datenbank-Verbindung wurde nicht explizit geschlossen (jetzt in der Müllabfuhr)
The connection was not closed by the application and is garbage collected [90018-170]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:175)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.th.nuernberg.itp.webservice.Database.Initalize(Database.java:19)
	at com.th.nuernberg.itp.webservice.BaseResource.createDatabaseInstance(BaseResource.java:36)
	at com.th.nuernberg.itp.webservice.DeviceResource.<init>(DeviceResource.java:22)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.sun.jersey.server.spi.component.ResourceComponentConstructor._construct(ResourceComponentConstructor.java:191)
	at com.sun.jersey.server.spi.component.ResourceComponentConstructor.construct(ResourceComponentConstructor.java:179)
	at com.sun.jersey.server.impl.resource.PerRequestFactory$PerRequest._getInstance(PerRequestFactory.java:182)
	at com.sun.jersey.server.impl.resource.PerRequestFactory$AbstractPerRequest.getInstance(PerRequestFactory.java:144)
	at com.sun.jersey.server.impl.application.WebApplicationContext.getResource(WebApplicationContext.java:238)
	at com.sun.jersey.server.impl.uri.rules.ResourceClassRule.accept(ResourceClassRule.java:83)
	at com.sun.jersey.server.impl.uri.rules.RightHandPathRule.accept(RightHandPathRule.java:147)
	at com.sun.jersey.server.impl.uri.rules.RootResourceClassesRule.accept(RootResourceClassesRule.java:84)
	at com.sun.jersey.server.impl.application.WebApplicationImpl._handleRequest(WebApplicationImpl.java:1469)
	at com.sun.jersey.server.impl.application.WebApplicationImpl._handleRequest(WebApplicationImpl.java:1400)
	at com.sun.jersey.server.impl.application.WebApplicationImpl.handleRequest(WebApplicationImpl.java:1349)
	at com.sun.jersey.server.impl.application.WebApplicationImpl.handleRequest(WebApplicationImpl.java:1339)
	at com.sun.jersey.server.impl.container.httpserver.HttpHandlerContainer.handle(HttpHandlerContainer.java:191)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.AuthFilter.doFilter(Unknown Source)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.ServerImpl$Exchange$LinkHandler.handle(Unknown Source)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.ServerImpl$Exchange.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: Die Datenbank-Verbindung wurde nicht explizit geschlossen (jetzt in der Müllabfuhr)
The connection was not closed by the application and is garbage collected [90018-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 38 more
10-31 19:48:41 jdbc[5]: null
org.h2.message.DbException: Die Datenbank-Verbindung wurde nicht explizit geschlossen (jetzt in der Müllabfuhr)
The connection was not closed by the application and is garbage collected [90018-170]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:175)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.th.nuernberg.itp.webservice.Database.Initalize(Database.java:19)
	at com.th.nuernberg.itp.webservice.BaseResource.createDatabaseInstance(BaseResource.java:36)
	at com.th.nuernberg.itp.webservice.DeviceResource.<init>(DeviceResource.java:22)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.sun.jersey.server.spi.component.ResourceComponentConstructor._construct(ResourceComponentConstructor.java:191)
	at com.sun.jersey.server.spi.component.ResourceComponentConstructor.construct(ResourceComponentConstructor.java:179)
	at com.sun.jersey.server.impl.resource.PerRequestFactory$PerRequest._getInstance(PerRequestFactory.java:182)
	at com.sun.jersey.server.impl.resource.PerRequestFactory$AbstractPerRequest.getInstance(PerRequestFactory.java:144)
	at com.sun.jersey.server.impl.application.WebApplicationContext.getResource(WebApplicationContext.java:238)
	at com.sun.jersey.server.impl.uri.rules.ResourceClassRule.accept(ResourceClassRule.java:83)
	at com.sun.jersey.server.impl.uri.rules.RightHandPathRule.accept(RightHandPathRule.java:147)
	at com.sun.jersey.server.impl.uri.rules.RootResourceClassesRule.accept(RootResourceClassesRule.java:84)
	at com.sun.jersey.server.impl.application.WebApplicationImpl._handleRequest(WebApplicationImpl.java:1469)
	at com.sun.jersey.server.impl.application.WebApplicationImpl._handleRequest(WebApplicationImpl.java:1400)
	at com.sun.jersey.server.impl.application.WebApplicationImpl.handleRequest(WebApplicationImpl.java:1349)
	at com.sun.jersey.server.impl.application.WebApplicationImpl.handleRequest(WebApplicationImpl.java:1339)
	at com.sun.jersey.server.impl.container.httpserver.HttpHandlerContainer.handle(HttpHandlerContainer.java:191)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.AuthFilter.doFilter(Unknown Source)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.ServerImpl$Exchange$LinkHandler.handle(Unknown Source)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.ServerImpl$Exchange.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: Die Datenbank-Verbindung wurde nicht explizit geschlossen (jetzt in der Müllabfuhr)
The connection was not closed by the application and is garbage collected [90018-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 38 more
11-01 12:13:58 jdbc[8]: null
org.h2.message.DbException: Die Datenbank-Verbindung wurde nicht explizit geschlossen (jetzt in der Müllabfuhr)
The connection was not closed by the application and is garbage collected [90018-170]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:175)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.th.nuernberg.itp.webservice.Database.Initalize(Database.java:19)
	at com.th.nuernberg.itp.webservice.BaseResource.createDatabaseInstance(BaseResource.java:36)
	at com.th.nuernberg.itp.webservice.DeviceResource.<init>(DeviceResource.java:22)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.sun.jersey.server.spi.component.ResourceComponentConstructor._construct(ResourceComponentConstructor.java:191)
	at com.sun.jersey.server.spi.component.ResourceComponentConstructor.construct(ResourceComponentConstructor.java:179)
	at com.sun.jersey.server.impl.resource.PerRequestFactory$PerRequest._getInstance(PerRequestFactory.java:182)
	at com.sun.jersey.server.impl.resource.PerRequestFactory$AbstractPerRequest.getInstance(PerRequestFactory.java:144)
	at com.sun.jersey.server.impl.application.WebApplicationContext.getResource(WebApplicationContext.java:238)
	at com.sun.jersey.server.impl.uri.rules.ResourceClassRule.accept(ResourceClassRule.java:83)
	at com.sun.jersey.server.impl.uri.rules.RightHandPathRule.accept(RightHandPathRule.java:147)
	at com.sun.jersey.server.impl.uri.rules.RootResourceClassesRule.accept(RootResourceClassesRule.java:84)
	at com.sun.jersey.server.impl.application.WebApplicationImpl._handleRequest(WebApplicationImpl.java:1469)
	at com.sun.jersey.server.impl.application.WebApplicationImpl._handleRequest(WebApplicationImpl.java:1400)
	at com.sun.jersey.server.impl.application.WebApplicationImpl.handleRequest(WebApplicationImpl.java:1349)
	at com.sun.jersey.server.impl.application.WebApplicationImpl.handleRequest(WebApplicationImpl.java:1339)
	at com.sun.jersey.server.impl.container.httpserver.HttpHandlerContainer.handle(HttpHandlerContainer.java:191)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.AuthFilter.doFilter(Unknown Source)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.ServerImpl$Exchange$LinkHandler.handle(Unknown Source)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.ServerImpl$Exchange.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: Die Datenbank-Verbindung wurde nicht explizit geschlossen (jetzt in der Müllabfuhr)
The connection was not closed by the application and is garbage collected [90018-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 38 more
11-01 12:13:58 jdbc[8]: null
org.h2.message.DbException: Die Datenbank-Verbindung wurde nicht explizit geschlossen (jetzt in der Müllabfuhr)
The connection was not closed by the application and is garbage collected [90018-170]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:175)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.th.nuernberg.itp.webservice.Database.Initalize(Database.java:19)
	at com.th.nuernberg.itp.webservice.BaseResource.createDatabaseInstance(BaseResource.java:36)
	at com.th.nuernberg.itp.webservice.DeviceResource.<init>(DeviceResource.java:22)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.sun.jersey.server.spi.component.ResourceComponentConstructor._construct(ResourceComponentConstructor.java:191)
	at com.sun.jersey.server.spi.component.ResourceComponentConstructor.construct(ResourceComponentConstructor.java:179)
	at com.sun.jersey.server.impl.resource.PerRequestFactory$PerRequest._getInstance(PerRequestFactory.java:182)
	at com.sun.jersey.server.impl.resource.PerRequestFactory$AbstractPerRequest.getInstance(PerRequestFactory.java:144)
	at com.sun.jersey.server.impl.application.WebApplicationContext.getResource(WebApplicationContext.java:238)
	at com.sun.jersey.server.impl.uri.rules.ResourceClassRule.accept(ResourceClassRule.java:83)
	at com.sun.jersey.server.impl.uri.rules.RightHandPathRule.accept(RightHandPathRule.java:147)
	at com.sun.jersey.server.impl.uri.rules.RootResourceClassesRule.accept(RootResourceClassesRule.java:84)
	at com.sun.jersey.server.impl.application.WebApplicationImpl._handleRequest(WebApplicationImpl.java:1469)
	at com.sun.jersey.server.impl.application.WebApplicationImpl._handleRequest(WebApplicationImpl.java:1400)
	at com.sun.jersey.server.impl.application.WebApplicationImpl.handleRequest(WebApplicationImpl.java:1349)
	at com.sun.jersey.server.impl.application.WebApplicationImpl.handleRequest(WebApplicationImpl.java:1339)
	at com.sun.jersey.server.impl.container.httpserver.HttpHandlerContainer.handle(HttpHandlerContainer.java:191)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.AuthFilter.doFilter(Unknown Source)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.ServerImpl$Exchange$LinkHandler.handle(Unknown Source)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.ServerImpl$Exchange.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: Die Datenbank-Verbindung wurde nicht explizit geschlossen (jetzt in der Müllabfuhr)
The connection was not closed by the application and is garbage collected [90018-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 38 more
11-01 12:13:58 jdbc[8]: null
org.h2.message.DbException: Die Datenbank-Verbindung wurde nicht explizit geschlossen (jetzt in der Müllabfuhr)
The connection was not closed by the application and is garbage collected [90018-170]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:175)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.th.nuernberg.itp.webservice.Database.Initalize(Database.java:19)
	at com.th.nuernberg.itp.webservice.BaseResource.createDatabaseInstance(BaseResource.java:36)
	at com.th.nuernberg.itp.webservice.DeviceResource.<init>(DeviceResource.java:22)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.sun.jersey.server.spi.component.ResourceComponentConstructor._construct(ResourceComponentConstructor.java:191)
	at com.sun.jersey.server.spi.component.ResourceComponentConstructor.construct(ResourceComponentConstructor.java:179)
	at com.sun.jersey.server.impl.resource.PerRequestFactory$PerRequest._getInstance(PerRequestFactory.java:182)
	at com.sun.jersey.server.impl.resource.PerRequestFactory$AbstractPerRequest.getInstance(PerRequestFactory.java:144)
	at com.sun.jersey.server.impl.application.WebApplicationContext.getResource(WebApplicationContext.java:238)
	at com.sun.jersey.server.impl.uri.rules.ResourceClassRule.accept(ResourceClassRule.java:83)
	at com.sun.jersey.server.impl.uri.rules.RightHandPathRule.accept(RightHandPathRule.java:147)
	at com.sun.jersey.server.impl.uri.rules.RootResourceClassesRule.accept(RootResourceClassesRule.java:84)
	at com.sun.jersey.server.impl.application.WebApplicationImpl._handleRequest(WebApplicationImpl.java:1469)
	at com.sun.jersey.server.impl.application.WebApplicationImpl._handleRequest(WebApplicationImpl.java:1400)
	at com.sun.jersey.server.impl.application.WebApplicationImpl.handleRequest(WebApplicationImpl.java:1349)
	at com.sun.jersey.server.impl.application.WebApplicationImpl.handleRequest(WebApplicationImpl.java:1339)
	at com.sun.jersey.server.impl.container.httpserver.HttpHandlerContainer.handle(HttpHandlerContainer.java:191)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.AuthFilter.doFilter(Unknown Source)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.ServerImpl$Exchange$LinkHandler.handle(Unknown Source)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.ServerImpl$Exchange.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: Die Datenbank-Verbindung wurde nicht explizit geschlossen (jetzt in der Müllabfuhr)
The connection was not closed by the application and is garbage collected [90018-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 38 more
11-01 12:13:58 jdbc[8]: null
org.h2.message.DbException: Die Datenbank-Verbindung wurde nicht explizit geschlossen (jetzt in der Müllabfuhr)
The connection was not closed by the application and is garbage collected [90018-170]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:175)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.th.nuernberg.itp.webservice.Database.Initalize(Database.java:19)
	at com.th.nuernberg.itp.webservice.BaseResource.createDatabaseInstance(BaseResource.java:36)
	at com.th.nuernberg.itp.webservice.DeviceResource.<init>(DeviceResource.java:22)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.sun.jersey.server.spi.component.ResourceComponentConstructor._construct(ResourceComponentConstructor.java:191)
	at com.sun.jersey.server.spi.component.ResourceComponentConstructor.construct(ResourceComponentConstructor.java:179)
	at com.sun.jersey.server.impl.resource.PerRequestFactory$PerRequest._getInstance(PerRequestFactory.java:182)
	at com.sun.jersey.server.impl.resource.PerRequestFactory$AbstractPerRequest.getInstance(PerRequestFactory.java:144)
	at com.sun.jersey.server.impl.application.WebApplicationContext.getResource(WebApplicationContext.java:238)
	at com.sun.jersey.server.impl.uri.rules.ResourceClassRule.accept(ResourceClassRule.java:83)
	at com.sun.jersey.server.impl.uri.rules.RightHandPathRule.accept(RightHandPathRule.java:147)
	at com.sun.jersey.server.impl.uri.rules.RootResourceClassesRule.accept(RootResourceClassesRule.java:84)
	at com.sun.jersey.server.impl.application.WebApplicationImpl._handleRequest(WebApplicationImpl.java:1469)
	at com.sun.jersey.server.impl.application.WebApplicationImpl._handleRequest(WebApplicationImpl.java:1400)
	at com.sun.jersey.server.impl.application.WebApplicationImpl.handleRequest(WebApplicationImpl.java:1349)
	at com.sun.jersey.server.impl.application.WebApplicationImpl.handleRequest(WebApplicationImpl.java:1339)
	at com.sun.jersey.server.impl.container.httpserver.HttpHandlerContainer.handle(HttpHandlerContainer.java:191)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.AuthFilter.doFilter(Unknown Source)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.ServerImpl$Exchange$LinkHandler.handle(Unknown Source)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.ServerImpl$Exchange.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: Die Datenbank-Verbindung wurde nicht explizit geschlossen (jetzt in der Müllabfuhr)
The connection was not closed by the application and is garbage collected [90018-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 38 more
11-01 12:13:58 jdbc[8]: null
org.h2.message.DbException: Die Datenbank-Verbindung wurde nicht explizit geschlossen (jetzt in der Müllabfuhr)
The connection was not closed by the application and is garbage collected [90018-170]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:175)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.th.nuernberg.itp.webservice.Database.Initalize(Database.java:19)
	at com.th.nuernberg.itp.webservice.BaseResource.createDatabaseInstance(BaseResource.java:36)
	at com.th.nuernberg.itp.webservice.DeviceResource.<init>(DeviceResource.java:22)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.sun.jersey.server.spi.component.ResourceComponentConstructor._construct(ResourceComponentConstructor.java:191)
	at com.sun.jersey.server.spi.component.ResourceComponentConstructor.construct(ResourceComponentConstructor.java:179)
	at com.sun.jersey.server.impl.resource.PerRequestFactory$PerRequest._getInstance(PerRequestFactory.java:182)
	at com.sun.jersey.server.impl.resource.PerRequestFactory$AbstractPerRequest.getInstance(PerRequestFactory.java:144)
	at com.sun.jersey.server.impl.application.WebApplicationContext.getResource(WebApplicationContext.java:238)
	at com.sun.jersey.server.impl.uri.rules.ResourceClassRule.accept(ResourceClassRule.java:83)
	at com.sun.jersey.server.impl.uri.rules.RightHandPathRule.accept(RightHandPathRule.java:147)
	at com.sun.jersey.server.impl.uri.rules.RootResourceClassesRule.accept(RootResourceClassesRule.java:84)
	at com.sun.jersey.server.impl.application.WebApplicationImpl._handleRequest(WebApplicationImpl.java:1469)
	at com.sun.jersey.server.impl.application.WebApplicationImpl._handleRequest(WebApplicationImpl.java:1400)
	at com.sun.jersey.server.impl.application.WebApplicationImpl.handleRequest(WebApplicationImpl.java:1349)
	at com.sun.jersey.server.impl.application.WebApplicationImpl.handleRequest(WebApplicationImpl.java:1339)
	at com.sun.jersey.server.impl.container.httpserver.HttpHandlerContainer.handle(HttpHandlerContainer.java:191)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.AuthFilter.doFilter(Unknown Source)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.ServerImpl$Exchange$LinkHandler.handle(Unknown Source)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.ServerImpl$Exchange.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: Die Datenbank-Verbindung wurde nicht explizit geschlossen (jetzt in der Müllabfuhr)
The connection was not closed by the application and is garbage collected [90018-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 38 more
11-01 19:44:39 jdbc[11]: null
org.h2.message.DbException: Die Datenbank-Verbindung wurde nicht explizit geschlossen (jetzt in der Müllabfuhr)
The connection was not closed by the application and is garbage collected [90018-170]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:175)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at com.th.nuernberg.itp.webservice.Database.Initalize(Database.java:19)
	at com.th.nuernberg.itp.webservice.BaseResource.createDatabaseInstance(BaseResource.java:36)
	at com.th.nuernberg.itp.webservice.DeviceResource.<init>(DeviceResource.java:22)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.sun.jersey.server.spi.component.ResourceComponentConstructor._construct(ResourceComponentConstructor.java:191)
	at com.sun.jersey.server.spi.component.ResourceComponentConstructor.construct(ResourceComponentConstructor.java:179)
	at com.sun.jersey.server.impl.resource.PerRequestFactory$PerRequest._getInstance(PerRequestFactory.java:182)
	at com.sun.jersey.server.impl.resource.PerRequestFactory$AbstractPerRequest.getInstance(PerRequestFactory.java:144)
	at com.sun.jersey.server.impl.application.WebApplicationContext.getResource(WebApplicationContext.java:238)
	at com.sun.jersey.server.impl.uri.rules.ResourceClassRule.accept(ResourceClassRule.java:83)
	at com.sun.jersey.server.impl.uri.rules.RightHandPathRule.accept(RightHandPathRule.java:147)
	at com.sun.jersey.server.impl.uri.rules.RootResourceClassesRule.accept(RootResourceClassesRule.java:84)
	at com.sun.jersey.server.impl.application.WebApplicationImpl._handleRequest(WebApplicationImpl.java:1469)
	at com.sun.jersey.server.impl.application.WebApplicationImpl._handleRequest(WebApplicationImpl.java:1400)
	at com.sun.jersey.server.impl.application.WebApplicationImpl.handleRequest(WebApplicationImpl.java:1349)
	at com.sun.jersey.server.impl.application.WebApplicationImpl.handleRequest(WebApplicationImpl.java:1339)
	at com.sun.jersey.server.impl.container.httpserver.HttpHandlerContainer.handle(HttpHandlerContainer.java:191)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.AuthFilter.doFilter(Unknown Source)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.ServerImpl$Exchange$LinkHandler.handle(Unknown Source)
	at com.sun.net.httpserver.Filter$Chain.doFilter(Unknown Source)
	at sun.net.httpserver.ServerImpl$Exchange.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: Die Datenbank-Verbindung wurde nicht explizit geschlossen (jetzt in der Müllabfuhr)
The connection was not closed by the application and is garbage collected [90018-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 38 more
11-01 20:57:16 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Syntax Fehler in SQL Befehl "ITP[*].T_DEVICE SELECT * FROM ITP.T_NOTIFICATION"; erwartet "INSERT, {"
Syntax error in SQL statement "ITP[*].T_DEVICE SELECT * FROM ITP.T_NOTIFICATION"; expected "INSERT, {"; SQL statement:
ITP.T_DEVICE SELECT * FROM ITP.T_NOTIFICATION [42001-170]
